src/en/installconfig-rack.md

```diff
@@ -38,6 +39,26 @@ immediately in the web UI and begin to sync with the primary controller:
 One reason to add extra rack controllers is to achieve DHCP high availability
 (DHCP HA).
 
+
+## Unregister a rack controller
+
+If you ever want to unregister a rack controller, which is probably something
+you would only do to an *extra* one you may have added, you will need to
+*delete* it from the region API server; there is no 'unregister' command.
+
+To do so, navigate to the 'Nodes' page and then the 'Controller' tab. Enter the
+controller's page by clicking on the machine you want to delete and select
+'Delete' from the dropdown (and then 'Go'). MAAS will do the right thing if the
+controller was used for DHCP HA.
+
+Although similar, this is not conceptually equivalent to deleting a MAAS node.
+Here, you are deleting a machine that is a part of MAAS itself.
+
+!!! Note: Such a machine will re-instate itself as a rack controller if it's
```

Comment:
I don't think 'in-situ' is clear or exact enough here. May be better to spell this out a bit more.
"Unless the software on this machine is removed, rebooting it will currently cause the machine to re-instate itself as a rack-server..." 
or whatever the case actually is.